lib/frixel_design_system/components/menu.ex

```diff
@@ -81,6 +81,69 @@ defmodule FrixelDesignSystem.Components.Menu do
     """
   end
 
+  @doc """
+  Renders a dropdown menu with a list of links.
+
+  ## Examples
+
+      <.dropdown_list label="Menu" type="primary" links={[
+        %{path: "/about", name: "About", visibility: :visible},
+        %{path: "/contact", name: "Contact", visibility: :visible}
+      ]} />
+
+  """
+  attr(:links, :list, required: true)
+  attr(:type, :string, default: "secondary")
```

Comment:
```suggestion
  attr(:type, :string, default: "secondary", values: ~w"primary secondary")
```
Ca permet d'indiquer ce qu'on attend pour éviter la confusion